source/common/http/http2/codec_impl.cc

```diff
@@ -73,24 +87,50 @@ class Http2ResponseCodeDetailValues {
     }
   }
 };
+const char* codec_strerror(int error_code) { return nghttp2_strerror(error_code); }
+#else
+  const absl::string_view errorDetails(int) const { return oghttp2_err_unknown_; }
+};
+const char* codec_strerror(int) { return "unknown_error"; }
```

Comment:
@birenroy is there an equivalent function on oghttp2 ?  The loss of fidelity is undesirable here.